.config/README.md

```diff
@@ -100,21 +120,45 @@ const config = async (env): Promise<Configuration> => {
 };
 
 export default config;
-
 ```
 
 #### 3. Update the `package.json` to use the new Webpack config
 
 We need to update the `scripts` in the `package.json` to use the extended Webpack configuration.
 
 **Update for `build`:**
+
 ```diff
 -"build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
 +"build": "webpack -c ./webpack.config.ts --env production",
 ```
 
 **Update for `dev`:**
+
 ```diff
 -"dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
 +"dev": "webpack -w -c ./webpack.config.ts --env development",
 ```
+
+### Configure grafana image to use when running docker
+
+By default `grafana-enterprise` will be used as the docker image for all docker related commands. If you want to override this behaviour simply alter the `docker-compose.yaml` by adding the following build arg `grafana_image`. 
+
+**Example:**
+
+```yaml
+version: '3.7'
+
+services:
+  grafana:
+    container_name: 'myorg-basic-app'
+    build:
+      context: ./.config
+      args:
+        grafana_version: ${GRAFANA_VERSION:-9.1.2}
+        grafana_image: ${GRAFANA_IMAGE:-grafana}
+```
+
+In this example we are assigning the environment variable `GRAFANA_IMAGE` to the build arg `grafana_image` with a default value of `grafana`. This will give you the possibility to set the value while running the docker-compose commands which might be convinent in some scenarios.
```

Comment:
```suggestion
In this example, we assign the environment variable `GRAFANA_IMAGE` to the build arg `grafana_image` with a default value of `grafana`. This will allow you to set the value while running the docker-compose commands, which might be convenient in some scenarios.
```

I don't think this line is needed, but here are a few suggestions if you want to keep the line.

Reply:
I'll also PR these changes to the upstream repo 😊